server/service/IndividualServicesService.js

```diff
@@ -524,31 +523,29 @@ function getAllApplicationList() {
 }
 
 
-var resolveHttpClient = exports.resolveHttpClientLtpUuidFromForwardingName =  function (forwardingName) {
+var resolveHttpClient = exports.resolveHttpClientLtpUuidFromForwardingName = function (forwardingName) {
   return new Promise(async function (resolve, reject) {
-    try{
-    let ForwardConstructName = await ForwardingDomain.getForwardingConstructForTheForwardingNameAsync(forwardingName)
-    if (ForwardConstructName === undefined) {
-    return null;
-    }
-  let LogicalTerminationPointlist;
-  let httpClientUuidList = [];
-  let ForwardConstructUuid = ForwardConstructName[onfAttributes.GLOBAL_CLASS.UUID]
-  let ListofUuid = await ForwardingConstruct.getFcPortListAsync(ForwardConstructUuid)
- for (let i = 0; i < ListofUuid.length; i++) {
- let PortDirection = ListofUuid[i][[onfAttributes.FC_PORT.PORT_DIRECTION]]
-if (PortDirection === FcPort.portDirectionEnum.OUTPUT) {
-LogicalTerminationPointlist = ListofUuid[i][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT]
-let httpClientUuid = await logicalTerminationPoint.getServerLtpListAsync(LogicalTerminationPointlist)
-let tcpClientUuid = await logicalTerminationPoint.getServerLtpListAsync(httpClientUuid[0])
-
-httpClientUuidList.push(httpClientUuid[0],LogicalTerminationPointlist,tcpClientUuid[0]);
-
-}
- }
-resolve(httpClientUuidList)
-    }catch(error){
-console.log(error)
+    try {
+      let uuidList = {};
+     
+      let forwardConstructName = await ForwardingDomain.getForwardingConstructForTheForwardingNameAsync(forwardingName)
+      if (forwardConstructName === undefined) {
+        return [];
+      }
+      let forwardConstructUuid = forwardConstructName[onfAttributes.GLOBAL_CLASS.UUID]
+      let listofUuid = await ForwardingConstruct.getFcPortListAsync(forwardConstructUuid)
+      for (let i = 0; i < listofUuid.length; i++) {
+        let portDirection = listofUuid[i][[onfAttributes.FC_PORT.PORT_DIRECTION]]
+        if (portDirection === FcPort.portDirectionEnum.OUTPUT) {
+          let operationClientUuid = listofUuid[i][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT]
+          let httpClientUuid = (await logicalTerminationPoint.getServerLtpListAsync(operationClientUuid))[0];
+          let tcpClientUuid = (await logicalTerminationPoint.getServerLtpListAsync(httpClientUuid))[0];
+          uuidList = { httpClientUuid, tcpClientUuid,operationClientUuid }
```

Comment:
Since object is used here, the for cycle doesn't make sense, because if there would be two output ports, you'd override the first value with the second.

Reply:
done